neo4j/session_with_context.go

```diff
@@ -521,7 +535,7 @@ func (s *sessionWithContext) Run(ctx context.Context,
 	s.autocommitTx = &autocommitTransaction{
 		conn: conn,
 		res: newResultWithContext(conn, stream, cypher, params, func() {
-			s.retrieveBookmarks(conn, runBookmarks)
+			s.retrieveBookmarks(nil, conn, runBookmarks)
```

Comment:
Why is the context being omitted here?

Reply:
Fixed in e2299cc5f46f2f6acd623af39a96bc950994ef70